packages/angular/projects/dev/src/app/button-group/static/icon-buttons/icon-button-group.ts

```diff
@@ -1,12 +1,12 @@
 /*
- * Copyright (c) 2016-2018 VMware, Inc. All Rights Reserved.
+ * Copyright (c) 2016-2020 VMware, Inc. All Rights Reserved.
  * This software is released under MIT license.
  * The full license information can be found in LICENSE in the root directory of this project.
  */
 import { Component } from '@angular/core';
 
 @Component({
-  selector: 'clr-icon-button-group-demo',
+  selector: 'cds-icon-button-group-demo',
```

Comment:
does this selector need to change? the `cds-icon-button` is a different component from the `cds-icon`

Reply:
Agree! Thanks!